### PR TITLE
Add user service account

### PR DESCRIPTION
**Pull request by adfost**
Adds a feature to allow service accounts to be created for another user

Screen Shot 2021-08-20 at 12 21 31 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Users/AddUserServiceAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        .then((res) => {
+          setAddSending(false);
+          closeModalAndRefresh(res);
+        })
+        .catch((err: ErrorResponseHandler) => {
+          setAddSending(false);
+          setModalErrorSnackMessage(err);
+        });
+    }
+  }, [
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/AddUserServiceAccount.tsx:88]: Please add `user` to dependency array

**Review by bexsoft — CHANGES_REQUESTED** [review]
Please assign the PR to yourself:
Screen Shot 2021-08-20 at 21 25 33

There is a strange issue, where after adding a couple of service accounts, if I try once more, system throws me : `original error -> (user_service_accounts.go:45: The specified user does not exist. (Specified user does not exist))` and logs me out:


https://user-images.githubusercontent.com/33497058/130307876-a5763cf0-6e15-4474-9804-adcc8792b079.mp4

File: portal-ui/src/screens/Console/Users/AddUserServiceAccount.tsx (hunk 1 above)
Comment (adfost) [reply portal-ui/src/screens/Console/Users/AddUserServiceAccount.tsx:88]: resolved, also I think this might have resolved the bug.

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — CHANGES_REQUESTED** [review]
Issue still persist after multiple tries

**Review by dvaldivia — CHANGES_REQUESTED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
Reviewing with Daniel, this issue comes when the selected user doesn't have a policy assigned. the error comes from MinIO as 401. This will be fixed un a further version

**Pull request merged**